frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -13,6 +13,38 @@ describe("collection permissions", () => {
     cy.server();
   });
 
+  describe("item management", () => {
+    Object.entries(PERMISSIONS).forEach(([permission, userGroup]) => {
+      context(`${permission} access`, () => {
+        userGroup.forEach(user => {
+          onlyOn(permission === "curate", () => {
+            describe(`${user} user`, () => {
+              beforeEach(() => {
+                cy.signIn(user);
+              });
+
+              it.skip("should be able to duplicate the dashboard without obstructions from the modal (metabase#15256)", () => {
+                cy.visit("/collection/root");
+                openEllipsisMenuFor("Orders in a dashboard");
+                cy.findByText("Duplicate this item").click();
+                cy.get(".Modal")
+                  .as("modal")
+                  .within(() => {
+                    cy.findByRole("button", { name: "Duplicate" })
+                      .should("not.be.disabled")
+                      .click();
+                    cy.findByText("Failed").should("not.exist");
```

Comment:
Will this wait around for "Failed" not showing? It seems like the button first turns disabled, when the request is initialized and then errors after it gets a return from the request.

Otherwise I'm guessing checking if the modal is closed is okay.

Reply:
This simply checks that the button is not disabled originally. When the click happens, that check is already in the history so it doesn't matter what comes after.